offset_manager_test.go

```diff
@@ -0,0 +1,305 @@
+package sarama
+
+import (
+	"testing"
+	"time"
+)
+
+var (
+	broker, coordinator *mockBroker
+	testClient          Client
+	config              *Config
+)
```

Comment:
I would prefer to avoid globals on principle. the init functions can just return the additional values